app/javascript/stylesheets/home.scss

```diff
@@ -0,0 +1,119 @@
+// scss
+@import "../../../node_modules/mappy-breakpoints/mappy-breakpoints.scss";
```

Comment:
**issue (blocking)**: it's better that you are now using breakpoints and color variables, but since you are repeating the declaration on every file it's still not solving the problem.
If at this point is too complicated to only have 1 css entry file you can create a config.scss where you put the import of mappy-breakpoints, the breakpoints and the colors, and then you import the config.scss file in every file. That way you have access to the same variables in all files.

Reply:
I already have done 1 css entry file but I forgot to push 😫